tests/flow/tests_setup/tests.sh

```diff
@@ -94,6 +86,17 @@ valgrind_summary() {
 
 #----------------------------------------------------------------------------------------------
 
+get_tests_data() {
```

Comment:
I don't mind having this right now, but it forces developers to have aws account to run this command. 
Please see https://stackoverflow.com/questions/273743/using-wget-to-recursively-fetch-a-directory-with-arbitrary-files-in-it if we can use `wget`  with aws HTTP.

Reply:
Done. Using recursive `wget` was not possible, so we got the desired behavior by using a file that specify all the relevant resources to download.